<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCheckable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkable</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckableAktif">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkable aktif mi?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRepeat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Repeat</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToggleRepeat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Repeat Degistir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDefault">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToggleDefault">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Default Degistir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCikis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cikis</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCikis</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
